atlasdb-client/build.gradle

```diff
@@ -15,8 +15,17 @@ schemas = [
 
 libsDirName = file('build/artifacts')
 
+task myTestJar(type: Jar) {
+    from sourceSets.test.output
+    classifier = 'tests'
+}
+
 configurations {
-    testArtifacts.extendsFrom testRuntime
+    testArtifacts
+}
+
+artifacts {
+    testArtifacts myTestJar
```

Comment:
Oh, so `testArtifacts.extendsFrom testRuntime` means that anything that takes a dependency on stuff in `testArtifacts` also takes a dependency on anything in `testRuntime`, which I don't think we ever needed in the first place (as attributed by the fact that all our tests still pass with this).